From b5c618b276838a2913347c2449874c77a5fdeda8 Mon Sep 17 00:00:00 2001
From: Xinyu Chen <xinyu.chen@freescale.com>
Date: Fri, 19 Oct 2012 11:07:49 +0800
Subject: [PATCH 079/281] ENGR00230245 imx6: add lights, sensor, power HAL
 into PACKAGE

Add lights.imx6, sensor.SABRESD/ARM2 and power.imx6 into
PRODUCT_PACKAGES For user build.

Change-Id: Ibaa204cfd3e0a1424227a78c9b4482239f3c7fb5
Signed-off-by: Xinyu Chen <xinyu.chen@freescale.com>
---
 imx6/imx6.mk | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 89a8ae5..45dd2e9 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -72,13 +72,15 @@ PRODUCT_PACKAGES += \
 
 # imx6 Hardware HAL libs.
 PRODUCT_PACKAGES += \
-	sensors.freescale			\
+	sensors.SABRESD				\
+	sensors.ARM2				\
 	overlay.imx6				\
-	lights.freescale			\
+	lights.imx6				\
 	gralloc.imx6				\
 	copybit.imx6				\
 	hwcomposer.imx6             		\
 	camera.imx6                 		\
+	power.imx6				\
 	magd
 
 # Bluetooth firmware files.
-- 
1.8.0

